examples/express-sample/bin/www

```diff
@@ -4,16 +4,16 @@
  * Module dependencies.
  */
 
-var app = require('../app');
-var debug = require('debug')('express-sample:server');
-var http = require('http');
+var app = require("../app");
+var debug = require("debug")("express-sample:server");
+var http = require("http");
 
 /**
  * Get port from environment and store in Express.
  */
 
-var port = normalizePort(process.env.PORT || '3000');
-app.set('port', port);
+var port = normalizePort(process.env.PORT || "3000");
```

Comment:
What if the user does not use `process.env.PORT` to set up Express? Should we be pulling this elsewhere, if possible?

Reply:
Because this is an example app, we should not have to worry about this? 🤔 We are using the standard way to set up ports I would think. Happy to be challenged! Such as, what other way would people use commonly :)

Reply:
`process.env.PORT || "3000"` works that if the user doesn't have a `$PORT` in their env vars, `3000` will be used by default.

Yes, agreed with @sumitramanga I guess this is standard express:

![image](https://github.com/MindscapeHQ/raygun4node/assets/2494376/9b38f834-e03e-4a3e-8106-98810f7458e6)